test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ViewEnginePathTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Internal
+{
+    public class ViewEnginePathTest
```

Comment:
Add another `[Theory]` with odd cases that trip up `RequiresPathResolution(...)` e.g. `/.hidden/views/index.cshtml`. `Assert.Equal()` should work fine there.

Reply:
Nice. These cases don't exist anymore.